app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.jsx

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { getAssetPath } from '@18f/identity-assets';
+import { FullScreen } from '@18f/identity-components';
+import AcuantContext from '../context/acuant';
+
+function FullScreenLoadingSpinner({ fullScreenRef, onRequestClose, fullScreenLabel }) {
+  return (
+    <FullScreen ref={fullScreenRef} label={fullScreenLabel} onRequestClose={onRequestClose}>
+      <img
+        src={getAssetPath('loading-badge.gif')}
+        alt=""
+        width="144"
+        height="144"
+        className="acuant-capture-canvas__spinner"
+      />
+    </FullScreen>
+  );
+}
+
+function AcuantSelfieCaptureCanvas({ fullScreenRef, onRequestClose, fullScreenLabel }) {
+  const { isReady } = useContext(AcuantContext);
+  return !isReady ? (
+    <FullScreenLoadingSpinner
+      fullScreenRef={fullScreenRef}
+      onRequestClose={onRequestClose}
+      fullScreenLabel={fullScreenLabel}
+    />
+  ) : (
+    <div id="acuant-face-capture-container" />
+  );
```

Comment:
Minor: In `if` / `else` conditions, I tend to suggest always putting the positive case first to avoid mental gymnastics of negating an else.

```suggestion
  return isReady ? (
    <div id="acuant-face-capture-container" />
  ) : (
    <FullScreenLoadingSpinner
      fullScreenRef={fullScreenRef}
      onRequestClose={onRequestClose}
      fullScreenLabel={fullScreenLabel}
    />
  );
```

Reply:
Makes sense, I was thinking that I wanted the <div> to feel like the default. Doing it this way is much clearer. Changed [here](https://github.com/18F/identity-idp/pull/9580/commits/541d507096af466cb04416403656b03f9b4ef960).